### PR TITLE
Added the new restiction for metrices name

### PR DESCRIPTION
**Pull request by LaithAlebrahim**
I've added condition to check the new rule and modified the ast names to pass the tests correctly.
Please review it and let me know if you have any concerns or you want to somehow change the logic or boolean variables name
Thanks @yegor256

### CODE REVIEW

File: tests/steps/test-measure.sh [hunk 1]
```diff
@@ -58,7 +58,18 @@ EOT
     metrics=$(find "${temp}" -name 'm1.*' -type f -exec basename {} \; | sort)
     echo -n "${metrics}" | while IFS= read -r m; do
         name=${m:3:100}
+        contains_special_case=false
+        correctly_formatted=false
         echo "Checking ${name}..."
+        if echo "${name}" | grep -E '(Mn|Mx|Av)'; then
```
Comment (yegor256) [inline tests/steps/test-measure.sh:64]: @LaithAlebrahim maybe it's better to do it this way:

```
if contains special case
  if ! correctly formatted
    print error
    exit 1
  fi
fi
```

**Comment by yegor256** [comment]
@LaithAlebrahim read this: https://www.yegor256.com/2024/04/01/ping-me-please.html

**Comment by LaithAlebrahim** [comment]
> @LaithAlebrahim read this: https://www.yegor256.com/2024/04/01/ping-me-please.html

@yegor256 ,Thanks for such a good blog.
I guess I followed the rules.
Did i miss any of rules unconsciously or it's just for more clarification. becasue when I made the changes I pingged you in the review and it seems it did not appear for you.
Also I already had been taught them by you during lecture ,and they were so helpful for me.
image
Thanks

**Comment by yegor256** [comment]
@LaithAlebrahim should I review the changes again? I don't understand

**Comment by LaithAlebrahim** [comment]
> @LaithAlebrahim should I review the changes again? I don't understand

Yes please I did what you asked me in the review here : https://github.com/yegor256/cam/pull/269/commits/4be5844e398b3fc3ec6481c59e4af07aad51bbe5

**Comment by LaithAlebrahim** [comment]
@yegor256  Yes please I did what you asked me to change in the review here : https://github.com/yegor256/cam/pull/269/commits/4be5844e398b3fc3ec6481c59e4af07aad51bbe5

**Comment by yegor256** [comment]
@LaithAlebrahim you don't need to use variables, I believe, check this out: https://www.yegor256.com/2015/09/01/redundant-variables-are-evil.html

**Comment by LaithAlebrahim** [comment]
> @LaithAlebrahim you don't need to use variables, I believe, check this out: https://www.yegor256.com/2015/09/01/redundant-variables-are-evil.html

@yegor256 yeah true ,Just got it ,I'll fix it very soon. Thanks

**Comment by LaithAlebrahim** [comment]
@yegor256 Done I removed the redundant varibales, could you please check it out now and review it. Thanks

**Comment by yegor256** [comment]
@LaithAlebrahim thanks!

**Pull request merged**